src/performance.ts

```diff
@@ -202,33 +215,46 @@ export class Performance {
-    debug('  • Commands Load Time: %sms', Performance.getResult('config.loadAllCommands')?.duration.toFixed(4) ?? 0)
-    debug('Core Plugin Load Time: %sms', Performance.highlights.corePluginsLoadTime.toFixed(4))
-    debug('User Plugin Load Time: %sms', Performance.highlights.userPluginsLoadTime.toFixed(4))
-    debug('Linked Plugin Load Time: %sms', Performance.highlights.linkedPluginsLoadTime.toFixed(4))
-    debug('Plugin Load Times:')
-    for (const [plugin, result] of Object.entries(Performance.highlights.pluginLoadTimes)) {
+    const oclifDebug = require('debug')('oclif-perf')
+    oclifDebug('Total Time: %sms', Performance.oclifPerf['oclif.runMs'].toFixed(4))
+    oclifDebug('Init Time: %sms', Performance.oclifPerf['oclif.initMs'].toFixed(4))
+    oclifDebug('Config Load Time: %sms', Performance.oclifPerf['oclif.configLoadMs'].toFixed(4))
+    oclifDebug('  • Plugins Load Time: %sms', Performance.getResult(OCLIF_MARKER_OWNER, 'config.loadAllPlugins')?.duration.toFixed(4) ?? 0)
+    oclifDebug('  • Commands Load Time: %sms', Performance.getResult(OCLIF_MARKER_OWNER, 'config.loadAllCommands')?.duration.toFixed(4) ?? 0)
+    oclifDebug('Core Plugin Load Time: %sms', Performance.oclifPerf['oclif.corePluginsLoadMs'].toFixed(4))
+    oclifDebug('User Plugin Load Time: %sms', Performance.oclifPerf['oclif.userPluginsLoadMs'].toFixed(4))
+    oclifDebug('Linked Plugin Load Time: %sms', Performance.oclifPerf['oclif.linkedPluginsLoadMs'].toFixed(4))
+    oclifDebug('Plugin Load Times:')
+    for (const [plugin, result] of Object.entries(Performance.oclifPerf.pluginLoadTimes)) {
       if (result.details.hasManifest) {
-        debug(`  ${plugin}: ${result.duration.toFixed(4)}ms`)
+        oclifDebug(`  ${plugin}: ${result.duration.toFixed(4)}ms`)
       } else {
-        debug(`  ${plugin}: ${result.duration.toFixed(4)}ms (no manifest!)`)
+        oclifDebug(`  ${plugin}: ${result.duration.toFixed(4)}ms (no manifest!)`)
       }
     }
 
-    debug('Hook Run Times:')
-    for (const [event, runTimes] of Object.entries(Performance.highlights.hookRunTimes)) {
-      debug(`  ${event}:`)
+    oclifDebug('Hook Run Times:')
+    for (const [event, runTimes] of Object.entries(Performance.oclifPerf.hookRunTimes)) {
+      oclifDebug(`  ${event}:`)
       for (const [plugin, duration] of Object.entries(runTimes)) {
-        debug(`    ${plugin}: ${duration.toFixed(4)}ms`)
+        oclifDebug(`    ${plugin}: ${duration.toFixed(4)}ms`)
       }
     }
 
-    debug('Command Load Time: %sms', Performance.highlights.commandLoadTime.toFixed(4))
-    debug('Command Run Time: %sms', Performance.highlights.commandRunTime.toFixed(4))
+    oclifDebug('Command Load Time: %sms', Performance.oclifPerf['oclif.commandLoadMs'].toFixed(4))
+    oclifDebug('Command Run Time: %sms', Performance.oclifPerf['oclif.commandRunMs'].toFixed(4))
+
+    const nonOclifDebug = require('debug')('non-oclif-perf')
```

Comment:
This is a misleading name. What if we end up putting a bunch of markers in `@oclif/plugin-update`?